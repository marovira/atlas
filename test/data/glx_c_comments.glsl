#version 450 core

/*
 * Some random comment
 */
layout(location = 0) in vec4 pos;

/*
 /*
 * Other comment nested.
 */

void main()
{
    gl_Position = pos;
}
