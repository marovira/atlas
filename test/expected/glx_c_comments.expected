#version 450 core
#line 2 0

/*
 * Some random comment
 */
#line 6 0
layout(location = 0) in vec4 pos;
#line 7 0

/*
 /*
 * Other comment nested.
 */
#line 12 0

#line 13 0
void main()
#line 14 0
{
#line 15 0
    gl_Position = pos;
#line 16 0
}
